src/main/java/com/kobylynskyi/graphql/codegen/model/definitions/ExtendedDefinition.java

```diff
@@ -14,6 +14,9 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static java.util.function.Predicate.not;
```

Comment:
Predicate#not was released only in Java 11: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/function/Predicate.html#not(java.util.function.Predicate)
Could you please make a backwards-compatible change that is supported by Java 8 as well?
Thanks.

Reply:
I took your suggestion and applied it!